tests/backup/backup_helper.go

```diff
@@ -1497,12 +1502,34 @@ func DeletePodWithLabelInNamespace(namespace string, label map[string]string) er
 	}
 
 	// fetch the newest set of pods post wait for pods to come up
-	pods, err = core.Instance().GetPods(namespace, label)
+	if ignoreLabel {
+		nolabel := make(map[string]string)
+		pods, err = core.Instance().GetPods(namespace, nolabel)
+	} else {
+		pods, err = core.Instance().GetPods(namespace, label)
+	}
 	if err != nil {
 		return err
 	}
-
 	for _, pod := range pods.Items {
+		flag := false
+		if ignoreLabel {
```

Comment:
Could you implement this logic as part of podList and keep the deletion for all pods fetched using same method irrespective of whether ignoreLabel is set or not